apps/frontend/src/pages/CreateListing.tsx

```diff
@@ -1,5 +1,281 @@
+                      id="field-description"
+                      label="Description"
+                      type={"text"}
+                      sx={{ m: "10px", display: "flex" }}
+                      rows={10}
+                      multiline
+                      onChange={(event) =>
+                        setNewListingObject((prevState) => ({
+                          ...prevState,
+                          listing: {
+                            ...prevState.listing,
+                            description: event.target.value,
+                          },
+                        }))
+                      }
+                    />
+                    <TextField
+                      id="field-price"
+                      label="Price(CAD)"
+                      type="number"
+                      sx={{ m: "10px" }}
+                      onChange={(event) =>
+                        setNewListingObject((prevState) => ({
+                          ...prevState,
+                          listing: {
+                            ...prevState.listing,
+                            price: parseFloat(event.target.value),
+                          },
+                        }))
+                      }
+                    />
+                  </FormControl>
+                </form>
+              </Box>
+              <Grid container sx={{ padding: "10px" }}>
+                <Grid item sm={12}>
+                  <Box sx={{ display: "flex" }}>
+                    <Button
+                      type="submit"
+                      variant="contained"
+                      sx={{
+                        display: "inline",
+                        mt: 2,
+                        backgroundColor: colors.martletplaceNavyBlue,
+                        "&:hover": {
+                          backgroundColor: colors.martletplaceBlueHover,
+                        },
+                        textTransform: "none",
+                        fontSize: "16px",
+                        padding: "10px 20px",
+                        margin: "10px",
+                      }}
+                      onClick={sendPostToCreateListing}
+                      id={"submit-button"}
+                    >
+                      Create Listing
+                    </Button>
+                    <MultiFileUpload
+                      passedImages={images}
+                      setPassedImages={setImages}
+                      multipleUpload={true}
+                      htmlForButton={buttonHTML}
+                    />
+                  </Box>
+                </Grid>
+                <Grid item sm={4}></Grid>
+              </Grid>
+            </Grid>
+            <Grid item lg={6} xs={12}>
+              <Box>
+                <Typography variant={"h5"} sx={{ paddingLeft: "20px" }}>
+                  Image Preview
+                </Typography>
+                <Box sx={{ padding: "10px" }}>
+                  {!isImageValid(images[0]) ? (
```

Comment:
Since isImageValid currently returns true on undefined, this might not work as expected